lib/ex_unit/lib/ex_unit/case.ex

```diff
@@ -223,7 +223,26 @@ defmodule ExUnit.Case do
     end
 
     quote do
-      async = !!unquote(opts)[:async]
+      @ex_unit_default_opts [async: true]
```

Comment:
You should be able to do this with a normal variable without the need for a module attribute.

Reply:
True, thanks for the heads up!

Reply:
Just realized the default value for async is actually wrong here, fixing with the new commits.